tests/test_settings.py

```diff
@@ -446,7 +453,20 @@ class Settings(BaseSettings):
 
         def _build_values(self, init_kwargs, _env_file, _env_file_encoding, _secrets_dir):
             config_settings = init_kwargs.pop('__config_settings__')
-            return {**config_settings, **init_kwargs, **self._build_environ()}
+            return {
```

Comment:
are you sure we still need to override `_build_values` here?

I suspect this case could now be accomplished with `customise_sources`.

Reply:
Totally, in the first place I kept the test this way to ensure the downstream custom config will still work. 
However, they can easily update their config using this new feature.